.github/workflows/nightly.yml

```diff
@@ -109,10 +109,11 @@ jobs:
             html \
             slides \
             linkcheck \
-            SPHINXOPTS="-Wn -A sidebar_version_name=${version}" \
+            SPHINXOPTS="-Wn --keep-going -A sidebar_version_name=${version}" \
```

Comment:
Nice. If a little weird - one might assume that --keep-going is the opposite of -W.